src/components/OSCALControlProse.js

```diff
@@ -384,6 +385,12 @@ export function OSCALReplacedProseWithByComponentParameterValue(props) {
   // https://github.com/EasyDynamics/oscal-react-library/issues/499
   const statementByComponentDescription =
     statementByComponent?.description || null;
+  const statementByComponentDescriptionMarkup = (
+    <OSCALMarkupMultiLine
+      text={statementByComponentDescription}
+      paragraphComponent="p"
+    />
+  );
```

Comment:
Should we do this in the case where `statementByComponentDescription` is `null`?

```tsx
const statementByComponentDescriptionMarkup = statementByComponentDescription ? (
  <OSCALMarkupMultiLine
    text={statementByComponentDescription}
    paragraphComponent="p"
  />
);
```

Reply:
That's a great idea, thank you!